data-transfer-hash/README.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: pattern
+title: Data Transfer Hash
+folder: data-transfer-hash
+permalink: /patterns/data-transfer-hash/
+categories: Architectural
+language: en
+tags:
+ - Structural
+ - Decoupling
```

Comment:
The pattern is listed under 'tier communications' category in the book 'J2EE design patterns'. I would put it under behavioral category with decoupling tag.